posts/intro-to-mathport.md

```diff
@@ -0,0 +1,131 @@
+  Instead the hope is that it gives us something that humans can plausibly improve to a complete translation of mathlib3.
+
+To understand how mathport (mostly talking about the synport part from here on) works,
+it's important to understand that it is translating mathlib3 to Lean 4 source code, "modulo" the current content of mathlib4.
+That is, the premise is that as we progressively construct mathlib4
+(whether by translating by hand, moving content from mathport's output to mathlib4, or adding `#align` statements)
+the output from running mathport on mathlib3 will change.
+In particular, as mathport is translating each declaration,
+it checks to see if a corresponding declaration in mathlib4 already exists, and is defeq.
+If so, mathport will instead just use that declaration.
+If not, mathport will make a copy, appending a `×` to the name.
+Sometimes these misalignments are due to an unintentional non-defeq, that can be fixed in mathlib4.
+Other times, we genuinely want to change something in mathlib4
+(e.g. to use Lean 4's multiple parent structures, which are better than `old_structure_cmd`).
+As a result, we expect that some misalignments will persist throughout the mathport-assisted stage of the port,
+and only afterwards will we polish these away.
+
+# What should I look at?
+
+Please note that mathport takes considerable resources to run on mathlib: approximately 3.5 hours, and 32gb of RAM.
+So you'll probably want to look at artifacts generated by CI rather than running it yourself.
+
+* https://github.com/leanprover-community/mathlib3port should be most people's first stop.
+  This contains a copy of the `.lean` files produced by a recent run of `mathport`, in the `Mathbin` directory.
+  You should just be able to check out of copy of this repository, and open the folder in VS Code,
+  to see the current state of mathport output.
+  You can also try out the above example with `#check AlgebraicGeometry.Scheme` in a fresh file here.
+  * Remember it's expected to be horribly broken; most tactics aren't implemented,
+    and there are bugs around parenthesization of arguments, and name resolution!
+  * Good luck finding even a single file that compiles cleanly right now.
+ * https://github.com/leanprover-community/lean3port is the corresponding repository
+   containing a copy of a recent run of `mathport` on the Lean 3 core library.
+   It's less interesting perhaps, but also smaller and easier to inspect.
+ * https://github.com/leanprover-community/mathport contains the code for mathport itself,
+   as well as the continuous integration set up that runs mathport on Lean 3 core and mathlib3
+   every time there is either a PR to mathport, or a commit to master.
+   The artifacts produced by CI appear at https://github.com/leanprover-community/mathport/releases,
+   and the two repositories listed above have a `lakefile.lean` that will download and unpack these artifacts.
+
```

Comment:
Add an item for mathlib4? If you're asking people to port tactics, this would be handy.

Reply:
I've added a quite large bullet point about `mathlib4`. Could you please read over it?